nc/data/download_from_nc.py

```diff
@@ -31,26 +38,20 @@ def nc_download_and_unzip_data(destination, prefix='state-'):
         logger.debug("{} exists, skipping download".format(zip_filename))
     else:
         logger.debug("Downloading data to {}".format(zip_filename))
-        nc_data_site = 'sbi1.jus.state.nc.us'
-        nc_data_user = os.environ.get('NC_FTP_USER')
-        nc_data_password = os.environ.get('NC_FTP_PASSWORD')
+        nc_data_site = os.environ.get('NC_FTP_HOST')
         nc_data_file = 'STOPS_Extract.zip'
         nc_data_directory = '/TSTOPextract'
-
-        # Note: NC documents show FileZilla set up to use explicit FTP over TLS
-        #       if available (like FTP_TLS), but the server doesn't currently
-        #       support it.
-        ftp = FTP(nc_data_site)
-        ftp.login(nc_data_user, nc_data_password)
-        ftp.cwd(nc_data_directory)
+        ftps = ftps_connect(nc_data_site)
+        ftps.prot_p()
```

Comment:
What's prot_p for?

Reply:
It actually secures the connection. Without this call the connection fails.  Maybe there is a way to integrate it into the tls constructor.

Reply:
https://docs.python.org/3/library/ftplib.html#ftplib.FTP_TLS

`Securing the data connection requires the user to explicitly ask for it by calling the prot_p() method. `